metaserver/src/main/resources/logback.groovy

```diff
@@ -0,0 +1,15 @@
+def defaultPattern = "%d{HH:mm:ss.SSS} %-5level %logger{20} - %msg%n"
+
+appender("STDOUT", ch.qos.logback.core.ConsoleAppender) {
+  encoder(PatternLayoutEncoder) {
+    pattern = defaultPattern
+  }
+}
+
+appender("LSP", scala.meta.languageserver.LSPLogger) {
```

Comment:
Out of curiosity, was this not possible in logback.xml?

Reply:
it was, but I suspect declaring re-usable variables (like `defaultPattern`) is a bit more clunky.
Overall I find the groovy configuration more readable.